docs/build/getting-started/quickstarts/testing.md

```diff
@@ -0,0 +1,102 @@
+
+# Introduction to Testing in Cadence
+
+Testing is an essential part of smart contract development to ensure the correctness and reliability of your code. In Cadence, the testing framework provides a convenient way to write tests for Cadence programs, allowing you to verify the functionality of your smart contracts.
+
+## Install Flow CLI
+
+The [Flow CLI](../../../tools/flow-cli/index.md) is the primary tool for developing, testing, and deploying smart contracts written in Cadence to the Flow network. You can install the Flow CLI by following the [installation instructions](../../../tools/flow-cli/install.md) provided by the Flow documentation.
+
+## Create a new project
+
+In your preferred code editor, create a new directory for your project and navigate to it in the terminal. Then 
+initialize a new Flow project by running the command `flow init`. This will create a `flow.json` file that contains the project configuration.
+```bash
+mkdir test-cadence
+cd test-cadence
+flow init
+```
+
+## Write a simple smart contract
+
+In your code editor, create a new file called `calculator.cdc` and add the following code:
+
+```cadence calculator.cdc
+access(all) contract Calculator {
+    access(all) fun add(a: Int, b: Int): Int {
+        return a + b;
+    }
+
+    access(all) fun subtract(a: Int, b: Int): Int {
+        return a - b;
+    }
+}
+```
+
+## Add the smart contract to the config
+
+Run `flow config add contract` and add the created `calculator.cdc` contract to the config with the name as `Calculator`:
```

Comment:
quick explainer: "This command adds your contract name and location under the contracts key in flow.config..."